README.md

```diff
@@ -99,38 +99,45 @@ In the **Console** type:
 This will return something like this: `<header class="site-
 header">...</header>`. Go ahead and click on that disclosure triangle to see
 more. This is the DOM Node, a JavaScript `object`. This means that it, in turn,
-can have methods called on it! Let's save our header to a `variable`.
-
-```javascript
-  var headerObj = document.querySelector('header')
-```
+can have methods called on it! This is called _method chaining_. Let's use
+_method chaining_ to remoe our node from the DOM.
 
 ### Delete an Element with JavaScript
 
 Now type:
 
-```javascript
-  headerObj.remove()
-```
-
-The header is gone! 
-
-If you refresh the page to get the `header` back, you could also try for some
-slightly advanced JavaScript:
-
 ```javascript
   document.querySelector('header').remove()
 ```
 
-This is called method chaining. Since the call of `querySelector()` to the
-document object returned a header object which supported a method called
-`remove()`, we used dot-notation to chain the calls.
+The header is gone! We called `document.querySelector('header')` in order
+to get the node onto which we _chained_ the call to `remove()`. We use
+dot-notation to _chain_ the calls.
 
 ### Demonstrate That the Source is Not Changed When the DOM Is
 
 Follow the same process we followed earlier to verify that the source has not
 changed. To restore it, simply refresh the page (i.e. reload the DOM).
 
+### Preview JavaScript Variables
+
+Have you ever heard a celebrity or royalty refer to themselves in the third
+person repeatedly:
+
+"Wanye Kest is the greatest rapper ever to walk this earth. When
+Wanye Kest is hungry, he likes grilled cheese sandwiches, but not too much
```

Comment:
"Wanye" is a little weird, it makes it so I as a reader have to keep track of two things 1. the point about referring to themselves in the third person 2. this translation